README_PYPI.md

```diff
@@ -0,0 +1,277 @@
+Pywbemtools is a collection of command line tools that communicate with
+WBEM servers. The tools are written in pure Python and support Python 2
+and Python 3.
+
+At this point, pywbemtools includes a single command line tool named
+`pywbemcli` that uses the
+[pywbem package on Pypi](https://pypi.org/project/pywbem/) to issue operations
+to a WBEM server using the
+[CIM/WBEM standards](https://www.dmtf.org/standards/wbem/) defined by the
+[DMTF](https://www.dmtf.org/) to perform system management tasks.
+
+CIM/WBEM standards are used for a wide variety of systems management
+tasks in the industry including DMTF management standards and the
+[SNIA](https://www.snia.org/) Storage Management Initiative Specification
+([SMI-S](https://www.snia.org/forums/smi/tech_programs/smis_home)).
+
+Pywbemcli provides access to WBEM servers from the command line. It
+provides functionality to:
+
+- Explore the CIM data of WBEM servers. It can manage/inspect the CIM
+  model components including CIM classes, CIM instances, and CIM
+  qualifiers and execute CIM methods and queries on the WBEM server.
+- Execute specific CIM-XML operations on the WBEM server as defined in
+  [DMTF](https://www.dmtf.org/) standard
+  [DSP0200 (CIM Operations over HTTP)](https://www.dmtf.org/sites/default/files/standards/documents/DSP0200_1.4.0.pdf).
+- Inspect and manage WBEM server functionality including:
+  - CIM namespaces
+  - Advertised WBEM management profiles
+  - WBEM server brand and version information
+- Capture detailed information on CIM-XML interactions with the WBEM
+  server including time statistics and details of data flow.
+- Maintain a file with persisted WBEM connection definitions so that
+  pywbemcli can access multiple WBEM servers by name.
+- Provide both a command line mode and an interactive mode where
+  multiple pywbemcli commands can be executed within the context of a
+  WBEM server.
+- Use an integrated mock WBEM server to try out commands. The mock
+  server can be loaded with CIM objects defined in MOF files or via
+  Python scripts.
+
+# Documentation
+
+For the latest version of pywbemtools released on Pypi:
+
+- [Documentation](https://pywbemtools.readthedocs.io/en/stable/)
+- [Installation](https://pywbemtools.readthedocs.io/en/stable/introduction.html#installation)
+- [Change history](https://pywbemtools.readthedocs.io/en/stable/changes.html)
+
+# Quickstart
+
+All commands within pywbemcli show help with the `-help` or `-h` options:
+
+``` text
+$ pywbemcli --help
+. . .
+$ pywbemcli connection --help
+. . .
+$ pywbemcli connection save --help
+. . .
+```
+
+The following examples build on each other and show a typical sequence
+of exploration of a WBEM server. For simplicity, they all operate
+against the default namespace of the server, and use a persistent
+connection definition for the server:
+
+-   Add a persistent connection definition named `conn1` for the WBEM
```

Comment:
change to say persistent WBEM server connection

Reply:
It already mentions "WBEM server" in the second half of the sentence:
```
Add a persistent connection definition named `conn1` for the WBEM
server to be used,
```